src/sonic-host-services/scripts/hostcfgd

```diff
@@ -793,6 +793,12 @@ class HostConfigDaemon:
                 syslog.syslog(syslog.LOG_WARNING, "Enable state of feature '{}' is None".format(feature_name))
                 continue
 
+            # dhcp_relay WA for production issue
```

Comment:
what do you mean by WA? to ensure upgrade and backwards compatibility?

Reply:
MSFT have a production issue, they want only ToRRouter switches to enable dhcp relay service.
The previous approach was to disable the feature by not hooking dhcp packets, this approach allow the user to enable it if it is configured to do so but still address the MSFT requirement to enable it by default only for ToRRouter.
If it is not a ToRRouter the user will have to override the configuration to enable it.